public/css/list/comment-list.less

```diff
@@ -13,7 +13,6 @@
 
   .title a {
     &.subject {
-      overflow: visible;
```

Comment:
Not sure about this. I've added this as I thought the username is the most important thing here. See:

**with overflow visible**
![Screenshot from 2021-11-04 09-40-40](https://user-images.githubusercontent.com/16668527/140283448-88d1dc36-e868-4913-88e0-18d0717f3896.png)

**without**
![Screenshot from 2021-11-04 09-40-50](https://user-images.githubusercontent.com/16668527/140283476-7ff91c8e-defc-4a75-a620-87002214b558.png)

Currently only the *commented* part is not visible, but with long usernames I believe they'll also get compressed too far so that they are not readable anymore. And I think those are more important than the object names, aren't they?

But yes, I know, veeeeery long usernames then cause overflow again or get cut...

Reply:
Your example is fine. But I wouldn’t consider this as accaptable.

![Screen Shot 2021-11-04 at 13 13 18](https://user-images.githubusercontent.com/6977434/140311641-9566b01c-6bb5-455f-8a11-8e98fd983d2d.png)

I suspect there won’t be an elegant CSS solution because `text-ellipsis` is very limited. As discussed we should rely on the detailed view mode to handle this.

The only alternative I see is having the title break in normal view-mode as well in minimal layout, but this would make the detailed view mode almost obsolete.

Reply:
Nah, as said, I'm not sure and don't have an alternative solution either..